docs/manage/_index.md

```diff
@@ -14,12 +14,19 @@ The [Viam app](https://app.viam.com/) is a web UI for managing and building robo
 ## Create Account and Log In
 
 To get started on the Viam app, you must log in as an authorized user.
+Viam support sign up using Google, GitHub, Apple, and Email.
+
 Navigate to [the main page](https://app.viam.com/).
-If you haven't already, click **Sign Up** to create a new user.
-Select **Log In** to log in to the app with your Google credentials, or provide your Email address and choose a password.
+If you haven't created an account yet, click **Sign Up** to create a new account using your prefered Single Sign On method or your email address and a password.
+If you already have an account, click **Log In** to log in using your Single Sign On credentials or your email address and password.
 
 If you forget your password to the app, click **Forgot password** and enter your email address to obtain instructions to reset your password.
 
+{{< alert title="Info" color="info" >}}
+We do not automatically merge Single Sign On accounts.
+If you use Google to log in and then sign up with Apple, Viam treats that as two separate accounts.
```

Comment:
Not a comment on wording here but this is fairly common right? If this is specifically requested would agree to keep but not sure if this really needs a tip

Reply:
Lots of places merge accounts and we will probably do that in the future. We'll remove this comment then :)

Reply:
I would just state that accounts created from separate auth sources are unique to each other, rather than stating that we don't do a thing the user was perhaps not assuming we did.

Q: Does it also apply equally to Email auth (non-SSO)? That is, if I have an Email/PW account and create an Apple SSO account -- are those separate as well? All previous language uses "SSO or Email" language, so I think we'd need mention here as well if so.

Reply:
Also separate - just tested